sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -675,7 +675,8 @@
                 "supportsHttpsTrafficOnly": true,
                 "allowBlobPublicAccess": false,
                 "networkAcls": "[if(equals(parameters('storageAccountBehindVNet'), 'true'), variables('networkRuleSetBehindVNet'), json('null'))]",
-                "isHnsEnabled": "[equals(parameters('set_up_feature_store'), 'true')]"
+                "isHnsEnabled": "[equals(parameters('set_up_feature_store'), 'true')]",
+                "minimumTlsVersion": "[if(equals(parameters('set_up_feature_store'), 'true'), 'TLS1_2', 'TLS1_0')]"
```

Comment:
I am wondering how existing TLS version configured. if non-featurestore storage bump to tls1.1 or others, they need to update here too.

Reply:
before this pr, non-featurestore workspace storage's minimum version is not configured, and the default minimum version is TLS 1.0